[#13711][Docs] Document YSQL function yb_is_local_table (#13712)
yugabyte/yugabyte-db: docs/content/preview/api/ysql/exprs/func_yb_is_local_table.md

---
title: yb_is_local_table() function [YSQL]
headerTitle: yb_is_local_table(oid)
linkTitle: yb_is_local_table()
description: Returns whether the given 'oid' is a table replicated only in the local region.
menu:
  preview:
    identifier: api-ysql-exprs-yb_is_local_table
    parent: api-ysql-exprs
type: docs
---

## Synopsis

> `yb_is_local_table` is a function used to find whether a table is pinned to the local region. It takes the OID (Object Identifier) of the table as an argument and looks for a tablespace associated with the table. It checks whether the tablespace is confined to the same region as that of the PostgreSQL backend processing the query. If so, it returns true, false otherwise.

Reviewer: Is this actually the _YugabyteDB backend_ here?

```suggestion
`yb_is_local_table` indicates whether a table is pinned to the local region. The function takes a table's OID (object identifier), and looks for a tablespace associated with that table. If the tablespace is confined to the same region as that of the PostgreSQL backend processing the query, the function returns `true`. Otherwise, the function returns `false`.
```

      --master_flags "placement_cloud=aws,placement_region=us-west-1,placement_zone=us-west-1c" \
      --tserver_flags "placement_cloud=aws,placement_region=us-west-1,placement_zone=us-west-1c"

    ./bin/yugabyted start                           \
      --base_dir=/home/yugabyte/<IP2>/yugabyte-data \
      --listen=<IP2>                                \
      --join=<IP1>                                  \
      --master_flags "placement_cloud=aws,placement_region=us-east-2,placement_zone=us-east-2c" \
      --tserver_flags "placement_cloud=aws,placement_region=us-east-2,placement_zone=us-east-2c"

    ./bin/yugabyted start                            \
      --base_dir=/home/yugabyte/<IP3>/yugabyte-data  \
      --listen=<IP3>                                 \
      --join=<IP1>                                   \
      --master_flags "placement_cloud=aws,placement_region=us-east-1,placement_zone=us-east-1a" \
      --tserver_flags "placement_cloud=aws,placement_region=us-east-1,placement_zone=us-east-1a"
    ```

1. Use [yb-admin](../../../../admin/yb-admin/) to specify the placement configuration to be used by the cluster as follows:

    ```sh

>     ./bin/yb-admin -master_addresses <IP1>:7100 modify_placement_info aws.us-west-1.us-west-1c:1,aws.    us-east-1.us-east-1a:1,aws.us-east-2.us-east-2c:1 3

Reviewer: ```suggestion
    ./bin/yb-admin -master_addresses <IP1>:7100 modify_placement_info aws.us-west-1.us-west-1c:1,aws.us-east-1.us-east-1a:1,aws.us-east-2.us-east-2c:1 3
```
